src/component/eLibraryIn42Box/ELibraryBook.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const ELibraryBook = ({ book, rank }) => {
+  return (
+    <Link className="elibrary__book__wrapper" to={`/info/${book.id}`}>
+      {rank ? <span className="elibrary__book__rank">{rank}</span> : null}
+      <img
+        className="elibrary__book__image"
+        src={book.image}
+        alt={book.title}
+      />
+      <p className="elibrary__book__title">{book.title}</p>
+      <p className="elibrary__book__author">{book.author}</p>
+      <div className="elibrary__book__icon">실물책</div>
+    </Link>
+  );
+};
```

Comment:
```suggestion
const ELibraryBook = ({ book: { id, title, author, image }, rank }) => {
  return (
    <Link className="elibrary__book__wrapper" to={`/info/${id}`}>
      {rank ? <span className="elibrary__book__rank">{rank}</span> : null}
      <img
        className="elibrary__book__image"
        src={image}
        alt={title}
      />
      <p className="elibrary__book__title">{title}</p>
      <p className="elibrary__book__author">{author}</p>
      <div className="elibrary__book__icon">실물책</div>
    </Link>
  );
};
```
인자를 한 단계 더 펼쳐도 좋을 것 같습니다!